src/app/services/date.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DateService {
+
+  constructor() { }
+
+  minutesToMiliseconds(minutes: number) {
```

Comment:
Can you add return type?

`minutesToMilliseconds(minutes: number): number {`

In typescript if you type everything you can understand quickly ;)